stats.go

```diff
@@ -375,7 +367,12 @@ func (h *histogram) RecordDuration(value time.Duration) {
 }
 
 func (h *histogram) Start() Stopwatch {
-	return histogramStopwatch{start: globalClock.Now(), histogram: h}
+	return NewStopwatch(globalClock.Now(), h)
+}
+
+func (h *histogram) RecordStopwatch(stopwatchStart time.Time) {
```

Comment:
This name seems specific to the use case of `Stopwatch` below, should we consider a more generic name like `RecordSince(time.Time)` which is similar to [time.Since](https://golang.org/pkg/time/#Since) in the standard library